frontend/src/components/dialog/Dialog.vue

```diff
@@ -1,103 +1,50 @@
-<template>
-    <vue-final-modal
-        v-slot="{ close }"
-        v-bind="$attrs"
-        classes="modal-container"
-        content-class="modal-content"
-    >
-        <div class="modal__content">
-            <div class="modal-header">
-                <h5 class="modal-title">{{ title }}</h5>
-                <button
-                    type="button"
-                    class="btn-close"
-                    data-dismiss="modal"
-                    aria-label="Close"
-                    @click="close"
-                ></button>
-            </div>
-            <div class="modal-body">
-                <p>{{ message }}</p>
-            </div>
-            <div class="modal-footer modal__action">
-                <button
-                    type="button"
-                    class="btn btn-danger"
-                    v-on:click="$emit('confirm', close)"
-                >
-                    {{ confirmText }}
-                </button>
-                <button type="button" class="btn btn-info" @click="close">
-                    {{ cancelText }}
-                </button>
-            </div>
-        </div>
-    </vue-final-modal>
-</template>
-
 <script setup inherit-attrs="false" lang="ts">
+import Dialog from 'primevue/dialog';
```

Comment:
I think we can put this file under the common folder, no need to keep it separated.

Reply:
Thanks, I'll move it to common folder.